lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb

```diff
@@ -0,0 +1,163 @@
+require 'fastlane/action'
+require_relative '../../helper/github_helper'
+
+module Fastlane
+  module Actions
+    class CreateReleaseBackmergePullRequestAction < Action
+      DEFAULT_BRANCH = 'trunk'.freeze
+
+      def self.run(params)
+        token = params[:github_token]
+        repository = params[:repository]
+        release_branch = params[:release_branch]
+        default_branch = params[:default_branch]
+        target_branches_param = params[:target_branches]
+        labels = params[:labels]
+        milestone_title = params[:milestone_title]
+
+        unless release_branch.start_with?('release/')
+          UI.user_error!('`release_branch` must start with `release/`')
+        end
+
+        if target_branches_param.include?(release_branch)
+          UI.user_error!('`target_branches` must not contain `release_branch`')
+        end
+
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: params[:github_token])
+        target_milestone = milestone_title.nil? ? nil : github_helper.get_milestone(repository, milestone_title)
+
+        target_branches = if target_branches_param.empty?
+                            determine_target_branches(release_branch.delete('release/'), default_branch)
+                          else
+                            target_branches_param
+                          end
+
+        pr_urls = []
+
+        target_branches.each do |target_branch|
+          Fastlane::Helper::GitHelper.checkout_and_pull(release_branch)
+
+          pr_urls << create_backmerge_pr(
+            token: token,
+            repository: params[:repository],
+            title: "Merge #{release_branch} into #{target_branch}",
+            head_branch: release_branch,
+            base_branch: target_branch,
+            milestone: target_milestone&.number,
+            labels: labels
+          )
+        end
+
+        pr_urls
```

Comment:
You already have `repository = params[:repository]` defined on line 11, so maybe you should use that instead of repeating `params[:repository]` on this line 42?

Also, this whole `pr_urls = []` + `pr_urls << …` + `return pr_urls` can be replaced by a `map` 😉 

```suggestion
        target_branches.map do |target_branch|
          Fastlane::Helper::GitHelper.checkout_and_pull(release_branch)

          create_backmerge_pr(
            token: token,
            repository: repository,
            title: "Merge #{release_branch} into #{target_branch}",
            head_branch: release_branch,
            base_branch: target_branch,
            milestone: target_milestone&.number,
            labels: labels
          )
        end
```